docs/user-tutorial/result-summary.md

```diff
@@ -0,0 +1,122 @@
+---
+id: result-summary
+---
+
+# Result Summary
+
+## Introduction
+
+This tool is to generate a readable summary report based on the raw benchmark results of single or multiple machines.
+
+## Usage
+
+1. [Install SuperBench](../getting-started/installation) on the local machine.
+
+2. Prepare the raw data, and rule file under the current path or somewhere on the local machine.
+
+3. After installing the Superbnech and the files are ready, you can start to generate the result summary automatically using  `sb result summary` command. The detailed command can be found from [SuperBench CLI](../cli).
+
+  ```
+  sb result summary --data-file ./results-summary.jsonl --rule-file ./rule.yaml --output-file-format md --output-dir ${output-dir}
+  ```
+
+4. After the command is finished, you can find the output result file named 'results_summary.md' under ${output_dir}.
+
+## Input
+
+The input mainly includes 2 files:
+
+ - **raw data**: jsonl file including multiple nodes' results automatically generated by SuperBench runner.
+
+    `Tips`: this file can be found at ${output-dir}/results-summary.jsonl after each successful run.
+
+ - **rule file**: It uses YAML format and defines how to generate the result summary including how to classify the metrics and what statistical methods (P50, mean, etc.) to use.
+
+### rule file
+
+This section describes how to write rules in **rule file**.
+
+The convention is the same with [SuperBench Config File](https://microsoft.github.io/superbenchmark/docs/superbench-config), please view it first.
+
+Here is an overview of the rule file structure:
+
+scheme:
+```yaml
+version: string
+superbench:
+  rules:
+    ${rule_name}:
+      statistics:
+          - ${statistic_name}
+      categories: string
+      aggregate: (optional)[bool|string]
+      metrics:
+        - ${benchmark_name}/regex
+        - ${benchmark_name}/regex
+```
+
+example:
+```yaml
+# SuperBench rules
+version: v0.4
+superbench:
+  rules:
+    kernel_launch:
+      statistics:
+        - mean
+        - p90
+        - min
+        - max
+      aggregate: True
+      categories: KernelLaunch
+      metrics:
+        - kernel-launch/event_overhead
+        - kernel-launch/wall_overhead
+    nccl:
+      statistics: mean
+      categories: NCCL
+      metrics:
+        - nccl-bw/allreduce_8388608_busbw
+    ib-loopback:
+      statistics: mean
+      categories: RDMA
+      metrics:
+        - ib-loopback/IB_write_8388608_Avg_\d+
+      aggregate: ib-loopback/IB_write_.*_Avg_(\d+)
+```
+
+This rule file describes the rules used for the result summary.
+
+They are firstly organized by the rule name, and each rule mainly includes several elements:
```

Comment:
`They are organized by the rule name and each rule mainly includes several elements: `

Reply:
updated